tool/lib/bert-canonical-generator.py

```diff
@@ -214,16 +234,23 @@ def construct_canonical(query, masks, current_index, replacement):
                         type=int,
                         default=500,
                         help='top-k candidates to return when generating adjectives')
+    parser.add_argument('--model-name-or-path',
+                        type=str,
+                        default='bert-large-uncased',
+                        help='The name of the model (e.g. bert-large-uncased) or the path to the directory where the model is saved.')
+    parser.add_argument('--is-paraphraser',
+                        action='store_true',
+                        help='If the model has been trained on a paraphrasing corpus')
```

Comment:
set the default from here, instead of on the function, so we don't need to pass additional options most of the time.

Reply:
The defaults are set in both places (`--is-paraphraser` is False by default when not specified), but I can remove them from the function.

Reply:
oh, i see. that's fine. just add the docstring, then let's merge this, so i can run the new experiments.